Formula/arangodb.rb

```diff
@@ -22,22 +23,24 @@ class Arangodb < Formula
   # with a unified CLI
   resource "starter" do
     url "https://github.com/arangodb-helper/arangodb.git",
+      tag:      "0.14.15",
       revision: "e32307e9ae5a0046214cb066355a8577e6fc4148"
   end
 
   def install
     ENV["MACOSX_DEPLOYMENT_TARGET"] = MacOS.version
 
     resource("starter").stage do
-      ENV["GOPATH"] = Dir.pwd + "/.gobuild"
       ENV["DOCKERCLI"] = ""
       # use commit-id as projectBuild
       commit = `git rev-parse HEAD`.chomp
       system "make", "deps"
-      system "go", "build", "-ldflags", "-X main.projectVersion=0.14.15 -X main.projectBuild=#{commit}",
-                            "-o", "arangodb",
-                            "github.com/arangodb-helper/arangodb"
-      bin.install "arangodb"
+      ldflags = %W[
+        -s -w
+        -X main.projectVersion=#{resource("starter").version}
+        -X main.projectBuild=#{commit}
+      ]
+      system "go", "build", *std_go_args, "-ldflags", ldflags.join(" "), "github.com/arangodb-helper/arangodb"
```

Comment:
I would try:

```suggestion
      system "go", "build", *std_go_args, "-ldflags", ldflags.join(" ")
```

Reply:
Nah, since the formula is using `go@1.13` we need explicitly set `GO111MODULE=on`.
Anyway, thanks for looking at it ❤️